### ISSUE TITLE
Database Files Not Found

### ISSUE THREAD

**Issue by emeraldau** [AzuraCast/AzuraCast#2284]
**Using Docker installation method**

No
Host Operating System

Ubuntu 16.4 Server
Describe the bug

I'm unable to update or re-install Azuracast. It states that SQLSTATE[HY200] Can't be found and some other files aren't able to be found

  An exception occurred in driver: SQLSTATE[HY000] [2002] No such file or dir  \n                                                                                                                                                               ectory                                                                       \                                                                                                                                                             n                                                                                                                                                                                                                                            \n\nIn PDOConnection.php line 31:\n                                                                                                                                                                                                                 \n  SQLSTATE[HY000] [2002] No such file or directory  \n                                                                                                                                                                                                                 \n\nIn PDOConnection.php line 27:\n                                                                                                                                                                                                                 \n  SQLSTATE[HY000] [2002] No such fil                                                                                                                                                             e or directory  \n                                                    \n\nazurac                                                                                                                                                             ast:setup [--update] [--load-fixtures] [--release]", "stderr_lines": ["", "In Ab                                                                                                                                                             stractMySQLDriver.php line 93:", "                                                                                                                                                                                                                                            ", "  An exception occurred in driver: SQLSTATE                                                                                                                                                             [HY000] [2002] No such file or dir  ", "  ectory                                                                                                                                                                                                                                    ", "                                                                                                                                                                                                                                            ", "", "In PDOConnection.php line 31:"                                                                                                                                                             , "                                                    ", "  SQLSTATE[HY000] [20                                                                                                                                                             02] No such file or directory  ", "                                                                                                                                                                                                                 ", "", "In PDOConnection.php line 27:", "                                                                                                                                                                                                                 ", "  SQLSTATE[HY000] [2002] No such file or directory  ", "                                                                                                                                                                                                                 ", "", "azuracast:setup [--u                                                                                                                                                             pdate] [--load-fixtures] [--release]"], "stdout": "\nAzuraCast Setup\n==========                                                                                                                                                             =====\n\nWelcome to AzuraCast. Please wait while some key dependencies of AzuraC                                                                                                                                                             ast are set up...\n * Environment: Production\n * Installation Method: Ansible\n                                                                                                                                                             \n ! [NOTE] Running in update mode.                                                                                                                                                                                                            \n\nMigrating Legacy Configuration\n------------------------------\n\nConfigur                                                                                                                                                             ation already set up.\n\nSetting Up InfluxDB\n-------------------\n\nDatabase cr                                                                                                                                                             eated.\nRetention policies updated.\nContinuous queries created.\nInfluxDB datab                                                                                                                                                             ases created.\n\nRunning Database Migrations\n---------------------------", "std                                                                                                                                                             out_lines": ["", "AzuraCast Setup", "===============", "", "Welcome to AzuraCast                                                                                                                                                             . Please wait while some key dependencies of AzuraCast are set up...", " * Envir                                                                                                                                                             onment: Production", " * Installation Method: Ansible", "", " ! [NOTE] Running i                                                                                                                                                             n update mode.                                               ", "", "Migrating L                                                                                                                                                             egacy Configuration", "------------------------------", "", "Configuration alrea                                                                                                                                                             dy set up.", "", "Setting Up InfluxDB", "-------------------", "", "Database cre                                                                                                                                                             ated.", "Retention policies updated.", "Continuous queries created.", "InfluxDB                                                                                                                                                              databases created.", "", "Running Database Migrations", "-----------------------                                                                                                                                                             ----"]}



Screenshots
![image](https://user-images.githubusercontent.com/34394420/72236551-799c4700-362b-11ea-8b37-ec77a158f0c1.png)
![image](https://user-images.githubusercontent.com/34394420/72236563-86b93600-362b-11ea-89ce-965827f9caab.png)

**Comment by SC2Mitch** [comment]
Having the same issue here.

**Comment by SlvrEagle23** [comment]
@SC2Mitch The _exact_ same issue? As in, it's failing with a "file not found" error, on an Ansible installation?

**Comment by CodeSteele** [comment]
Grabbing this to test fresh installs of docker/Ansible.

**Comment by CodeSteele** [comment]
Couldn't reproduce on a clean and up-to-date 18.04 install, going to check 16.04 as per the original issue.

**Comment by CodeSteele** [comment]
Doesn't happen on 16.04 too, so related to an upgrade-only path.

**Comment by emeraldau** [comment]
Hey, Sorry I'll explain a bit more. Was quite rushed when I sent the message. So all of the sudden, my Azuracast went down, I think this was after the VPS was down for maintenance. Then once it was back up and running I got this page (
![image](https://user-images.githubusercontent.com/34394420/72340131-c1020080-371b-11ea-9fa8-9bcac6c79925.png)). Then I tried to do an update, thinking that'd help and then all of the sudden, that error message happened. Quite confused atm
